src/modules/profile/profile.service.ts

```diff
@@ -41,4 +41,8 @@ export class ProfileService {
 
     return { profileId: response.identifiers[0].id };
   }
+
+  async findOne(userId: number) {
+    return this.profileRepository.findOne({ where: { userId } });
+  }
```

Comment:
Since there is a findOneByUserId method in the repository, you call it here.

Reply:
Nice said, done.